web/annotation_layer_builder.css

```diff
@@ -53,6 +53,7 @@
       outline: calc(1.5px * var(--scale-factor)) solid CanvasText !important;
       background-color: ButtonFace !important;
       color: ButtonText !important;
+      user-select: text;
```

Comment:
Is seems that this rule only applies to `forced-color` mode, given where its placed, which probably wasn't intended since it's no longer possible to select in a Popup?

Reply:
Whops you are right, I now moved it to the right place.